admin/products_price_manager.php

```diff
@@ -407,157 +366,137 @@
+          ];
         }
         ?>
-        <?php if (isset($pInfo->products_id) && $pInfo->products_id != '' || isset($sInfo->products_id) && $sInfo->products_id != '') { ?>
+        <?php if (isset($pInfo->products_id) && $pInfo->products_id != '') { ?>
           <script>
-            var tax_rates = new Array();
+            let tax_rates = [];
     <?php
-    for ($i = 0, $n = sizeof($tax_class_array); $i < $n; $i++) {
-      if ($tax_class_array[$i]['id'] > 0) {
-        echo 'tax_rates["' . $tax_class_array[$i]['id'] . '"] = ' . zen_get_tax_rate_value($tax_class_array[$i]['id']) . ';' . "\n";
+    foreach ($tax_class_array as $key => $value) {
+      if ($key === 0) {
+        continue;
       }
+      echo '    tax_rates[' . $value['id'] . '] = ' . zen_get_tax_rate_value($value['id']) . ';' . "\n";
     }
     ?>
 
             function doRound(x, places) {
-                return Math.round(x * Math.pow(10, places)) / Math.pow(10, places);
+              return Math.round(x * Math.pow(10, places)) / Math.pow(10, places);
             }
 
             function getTaxRate() {
-                var selected_value = document.forms["new_prices"].products_tax_class_id.selectedIndex;
-                var parameterVal = document.forms["new_prices"].products_tax_class_id[selected_value].value;
-                if ((parameterVal > 0) && (tax_rates[parameterVal] > 0)) {
-                    return tax_rates[parameterVal];
-                } else {
-                    return 0;
-                }
+              const parameterVal = $('select[name="products_tax_class_id"]').val();
+              if ((parameterVal > 0) && (tax_rates[parameterVal] > 0)) {
+                return tax_rates[parameterVal];
+              } else {
+                return 0;
+              }
             }
-          </script>
-        <?php } ?>
-        <?php if (isset($pInfo->products_id) && $pInfo->products_id != '') { ?>
-          <script>
-            var ProductStartDate = new ctlSpiffyCalendarBox("ProductStartDate", "new_prices", "product_start", "btnDate1", "<?php echo (($pInfo->products_date_available <= '0001-01-01') ? '' : zen_date_short($pInfo->products_date_available)); ?>", scBTNMODE_CUSTOMBLUE);
-          </script>
-          <script>
             function updateGross() {
-                var taxRate = getTaxRate();
-                var grossValue = document.forms["new_prices"].products_price.value;
-                if (taxRate > 0) {
-                    grossValue = grossValue * ((taxRate / 100) + 1);
-                }
+              const taxRate = getTaxRate();
+              let grossValue = $('input[name="products_price"]').val();
+              if (taxRate > 0) {
+                grossValue = grossValue * ((taxRate / 100) + 1);
+              }
 
-                document.forms["new_prices"].products_price_gross.value = doRound(grossValue, 4);
+              $('input[name="products_price_gross"]').val(doRound(grossValue, 4));
```

Comment:
I'm curious: You converted some of these to jQuery syntax, but left others alone. 
Was there a reason?